### PR TITLE
Add command line interface to predict and plot a single peptide spectrum

### PR DESCRIPTION
**Pull request by RalfG**
With `ms2pip-single-prediction` a single peptide spectrum can be predicted with MS²PIP and plotted with [spectrum_utils](https://spectrum-utils.readthedocs.io/). For instance:

```sh
ms2pip-single-prediction "PGAQANPYSR" "-" 3 --model TMT
```

will write the following image to a file:

PGAQANPYSR-3-TMT

### CODE REVIEW

**Comment by RalfG** [comment]
Tests fail because rdkit cannot be imported (cannot be installed with pip). See bittremieux/spectrum_utils#14

**Comment by RalfG** [comment]
RDKit issue has been fixed.

File: ms2pip/single_prediction.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        """Get spectrum_utils.PeptideFragmentAnnotation objects."""
+        return [
+            sus.PeptideFragmentAnnotation(1, mz, annotation[0], annotation[1:])
+            for mz, annotation in zip(mz, annotation)
+        ]
+
+
+def _flatten(nested_list):
+    """Flatten a nested list by one level."""
+    return [item for sublist in nested_list for item in sublist]
```
Comment (paretje) [inline ms2pip/single_prediction.py]: Would `itertools.chain.from_iterable` be an alternative?

**Review by paretje — CHANGES_REQUESTED** [review]

File: ms2pip/single_prediction.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
```
Comment (paretje) [inline ms2pip/single_prediction.py]:
```suggestion
```

**Review by paretje — CHANGES_REQUESTED** [review]

**Review by paretje — APPROVED** [review]

**Pull request merged**